### PR TITLE
CB-169: Allow users to be searched

### PR DESCRIPTION
**Pull request by dufferzafar**
JIRA Issue: [CB-169](http://tickets.musicbrainz.org/browse/CB-169)

Screenshot:

search_users

### CODE REVIEW

File: critiquebrainz/frontend/views/search.py [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
+from critiquebrainz.data.model.user import User
 
 search_bp = Blueprint('search', __name__)
 
 RESULTS_LIMIT = 10
 
 
+def search_users(query, limit=None, offset=None):
+    users, count = User.list(display_name=query, limit=limit, offset=offset,
+                             sort='name', is_blocked=False)
```
Comment (dufferzafar) [inline critiquebrainz/frontend/views/search.py]: I am not too sure about these two (sort, is_blocked) options!

**Comment by gentlecat** [comment]
As I said to you on IRC, I'd like to move away from the ORM. So if you modify things or write new functions for database access, just write your own SQL query.

**Comment by gentlecat** [comment]
I'm actually not sure if we need a feature like this. We need to discuss it.

**Comment by gentlecat** [comment]
I'm closing this for now.

**Pull request closed without merging**